showcase/tests/integration/components/hds/app-frame/index-test.js

```diff
@@ -74,6 +74,24 @@ module('Integration | Component | hds/app-frame/index', function (hooks) {
     `);
     assert.dom('#test-app-frame-header').doesNotExist();
   });
+
+  // @isHeaderFixed hds-app-frame--is-header-fixed
+  test('it should have the hds-app-frame--is-header-fixed class by default', async function (assert) {
+    await render(hbs`
+        <Hds::AppFrame id="test-app-frame" />
+      `);
+    assert.dom('#test-app-frame').hasClass('hds-app-frame--is-header-fixed');
+  });
+
+  test('it should not add the hds-app-frame--is-header-fixed class when @isHeaderFixed is false', async function (assert) {
```

Comment:
probably I would test instead of `false` the case when `isHeaderFixed` is not set

Reply:
I clarified the wording